docs/resources/kibana_alerting_rule.md

```diff
@@ -86,8 +86,21 @@ Required:
 
 Optional:
 
+- `frequency` (Block List, Max: 1) The parameters for the action, which are sent to the connector. (see [below for nested schema](#nestedblock--actions--frequency))
```

Comment:
How do you feel about using the text from [here](https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-createruleid#operation-createruleid-body-application-json-elastic-api-version-2023-10-31-actions-frequency)?